packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -74,8 +80,7 @@ class S78Journey extends Journey {
 					response.answers &&
 						response.answers[questions.screeningOpinionEnvironmentalStatement.fieldName] === 'yes'
 				)
-				.addQuestion(questions.developmentDescription)
-				.addQuestion(questions.sensitiveArea),
```

Comment:
This was in this section twice, causing you to get stuck in a loop

Reply:
Good catch. Probably wouldn't get caught until next sprint when they do navigation tickets.